src/Core/Services/Implementations/OrganizationService.cs

```diff
@@ -392,9 +395,9 @@ public async Task<string> AdjustSeatsAsync(Guid organizationId, int seatAdjustme
             // Retain original collection method
             var collectionMethod = sub.CollectionMethod;
 
-            var subResponse = await subscriptionService.UpdateAsync(sub.Id, new SubscriptionUpdateOptions
+            var subUpdateOptions = new SubscriptionUpdateOptions()
```

Comment:
remove empty parens `()` at end of object constructor when using simplified object initialization (`{ ... }`).